test/parallel/test-vm-indexed-properties.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const vm = require('vm');
+
+const code =
```

Comment:
Instead of `+`-ing several strings, we can use one multi-line string by using backticks. 

```
 const code = `Object.defineProperty(this, 99, {  
      value: 20,  
      enumerable: true  
 });`; 
```

It's less characters total and you can copy and paste such strings into the REPL without removing all the `' + '` in between.